docker-compose.multinode.evm.yml

```diff
@@ -48,3 +48,10 @@ services:
     volumes:
       - "${NETWORK_NODE_LOGS_ROOT_PATH}/accountBalances/:/node/streams/accountBalances/"
       - "${NETWORK_NODE_LOGS_ROOT_PATH}/recordStreams/:/node/streams/recordstreams/"
+    networks:
+      - multinode-importer-evm-network
+
+networks:
+  multinode-importer-evm-network:
+    name: hedera-multinode-importer-evm
```

Comment:
do we need a different network here

Reply:
I guess not, I've replaced the extra networks with 
```
    network_mode: none
```
 declarations.